packages/client/src/generation/defaults/defaultBrowserIndex.ts

```diff
@@ -0,0 +1,13 @@
+export const defaultBrowserIndex = `class PrismaClient {
+    constructor() {
+      throw new Error(
+        \`@prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
+  In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues/new\`,
+      )
+    }
+  }
+  
+  module.exports = {
+    PrismaClient,
+  }
+`
```

Comment:
Instead of having files that we copy around, we have file contents that we write to disk.